rskj-core/src/main/java/co/rsk/peg/federation/FederationMember.java

```diff
@@ -154,4 +161,30 @@ public int hashCode() {
                 mstPublicKey
         );
     }
+
+    public byte[] serialize() {
+        byte[][] rlpElements = new byte[LIST_SIZE][];
+        byte[] btcPublicKeyBytes = this.getBtcPublicKey().getPubKeyPoint().getEncoded(true);
+        byte[] rskPublicKeyBytes = this.getRskPublicKey().getPubKey(true);
+        byte[] mstPublicKeyBytes = this.getMstPublicKey().getPubKey(true);
+
+        rlpElements[BTC_KEY_INDEX] = RLP.encodeElement(btcPublicKeyBytes);
+        rlpElements[RSK_KEY_INDEX] = RLP.encodeElement(rskPublicKeyBytes);
+        rlpElements[MST_KEY_INDEX] = RLP.encodeElement(mstPublicKeyBytes);
+        return RLP.encodeList(rlpElements);
+    }
+
+    public static FederationMember deserialize(byte[] data) {
+        RLPList rlpList = (RLPList)RLP.decode2(data).get(0);
+
+        if (rlpList.size() != FEDERATION_RLP_LIST_SIZE) {
+            throw new RuntimeException(String.format("Invalid serialized FederationMember. Expected %d elements but got %d", LIST_SIZE, rlpList.size()));
+        }
+
+        BtcECKey btcKey = BtcECKey.fromPublicOnly(rlpList.get(BTC_KEY_INDEX).getRLPData());
```

Comment:
Same as in `serialize` method, I'd split this. `rlpList.get` and `BtcECKey.fromPublicOnly`. So we don't have many function calls in a single line